### DISCUSSION TITLE
Object exclude Area

### DISCUSSION THREAD

**Discussion by Zimbo111** [OrcaSlicer/OrcaSlicer#4304] [Ideas]
Is it possible if I specify an area of my print bed as a non-printable surface in the orca slicer to tell the slicer not to arrange in this area when auto-arranging? So far I only get an error message if there is an object in the area.
If you can simply exclude the area(s) from the print bed for arranging.

**Comment by MVerBerkmoes** [comment]
Yup .. look under Print Settings -> Basic Information -> Exclude Bed Area.
